core/arch/disassemble_shared.c

```diff
@@ -159,6 +159,12 @@ dsts_first(void)
     return TESTANY(DR_DISASM_INTEL | DR_DISASM_ARM, DYNAMO_OPTION(disasm_mask));
 }
 
+static inline bool
```

Comment:
Maybe the DR style should keep the braces around the k1?

Reply:
If you print the mask operand as a regular source, I don't see how this helps. If it was just up to me :) I would actually print the mask behind the destination, even for DR. WDYT?

Reply:
I put the {} back. At least it's consistent.

Reply:
> I would actually print the mask behind the destination, even for DR

That would be very confusing with the `->` separating sources and destinations.  To me that would only make sense if you removed the k register as a separate source and created some kind of masked-destination single operand that combined the destination and mask (much like the base reg of a mem ref is a dest even though that reg is not written to).